charts/alpha/memcached/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.tgz
+manifest.yml
```

Comment:
why do we need these? i think https://github.com/palantir/fedstart-helm-charts/blob/develop/.gitignore has you covered? 

Not opposed in general, but want to understand why this is here.